code/__HELPERS/filters.dm

```diff
@@ -22,15 +22,13 @@ GLOBAL_LIST_INIT(master_filter_info, list(
 			"size" = 1
 		)
 	),
-	/* Not supported because making a proper matrix editor on the frontend would be a huge dick pain.
-		Uncomment if you ever implement it
+	// Not implenmented, but if this isn't uncommented some windows will just error
```

Comment:
`implemented`
but also, this comment lacks context without the old one

Reply:
Not implemented. Will detail though
```suggestion
	// Not implenmented, but if this isn't uncommented some windows will just error
	// Needs either a proper matrix editor, or just a hook to our existing one 
	// Issue is filterrific assumes variables will have the same value type if they share the same name, which this violates
	// Gotta refactor this sometime
```

Reply:
i meant as in you made a typo

Reply:
```suggestion
	// Not implemented, but if this isn't uncommented some windows will just error
```